doc/source/cluster/kubernetes/getting-started.ipynb

```diff
@@ -244,6 +258,163 @@
     "Note that in production scenarios, you will want to use larger Ray pods. In fact, it is advantageous to size each Ray pod to take up an entire Kubernetes node. See the [configuration guide](kuberay-config) for more details."
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "ef55f61f",
+   "metadata": {},
+   "source": [
+    "### Deploying a static Ray cluster\n",
+    "\n",
+    "\n",
+    "#### Deploying a static Ray cluster without KubeRay\n",
+    "\n",
+    "In this section, we will deploy a static Ray cluster into the default namespace without using KubeRay. To use a non-default namespace, specify the namespace in your kubectl commands:\n",
```

Comment:
```suggestion
    "In this section, we will deploy a static Ray cluster into the `default` namespace without using KubeRay. To use another namespace, specify the namespace in your kubectl commands:\n",
```

Reply:
Could you also make this change in [the previous subsection](https://github.com/ray-project/ray/pull/31403/files#diff-7480f682fe8e63c4e013a96b1e2e79c7586e6617818eebf5ddd128893498e825R161)?